### PR TITLE
docs: Add Alert, Banner and PaymentMethods docs

### PR DESCRIPTION
**Pull request by carolinamenezes**
## What's the purpose of this pull request?
Add Alert, Banner and PaymentMethods docs

### CODE REVIEW

File: docs/docs/reference/ui/molecules/Alert.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import PropsSection from '@site/src/components/PropsSection/PropsSection'
+
+# Alert
+
+Alerts display short messages related to the behavior of a system, feature or page.
+
+## Import
+
+```tsx
+import { Banner, BannerImage, BannerContent, BannerLink } from '@faststore/ui'
```
Comment (tlgimenes) [inline docs/docs/reference/ui/molecules/Alert.mdx:10]: 👀

Comment (carolinamenezes) [reply docs/docs/reference/ui/molecules/Alert.mdx:10]:
```suggestion
import { Alert } from '@faststore/ui'
```

Comment (carolinamenezes) [reply docs/docs/reference/ui/molecules/Alert.mdx:10]: Ops

File: docs/docs/reference/ui/molecules/Banner.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+<PropsSection name="Banner" />
+
+## Customization
+
+`data-store-breadcrumb`
+
+`data-breadcrumb-item`
+
+`data-breadcrumb-item-active`
```
Comment (carolinamenezes) [inline docs/docs/reference/ui/molecules/Banner.mdx:59]:
```suggestion
`data-store-banner`

`data-store-banner=('vertical'|'horizontal')`

`data-store-banner-image`

`data-store-banner-content`

`data-store-banner-link`
```

**Review by tlgimenes — APPROVED** [review]

**Pull request merged**